src/runtime/dom-particle-base.ts

```diff
@@ -8,11 +8,12 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {assert} from '../platform/assert-web.js';
-
 import {Entity} from './entity.js';
 import {BigCollection, Collection, Variable} from './handle.js';
 import {Particle} from './particle.js';
+import {SlotProxy} from './slot-proxy.js';
+
+export type RenderModel = object;
```

Comment:
Might be worth adding a comment explaining why this is such a broad / loose type. I was confused when I first ran into it.